crates/sncast/src/starknet_commands/helpers/fee.rs

```diff
@@ -1,9 +1,215 @@
-use clap::Args;
+use anyhow::{bail, ensure, Result};
+use clap::{Args, ValueEnum};
 use starknet::core::types::FieldElement;
 
 #[derive(Args, Debug)]
-pub struct Fee {
-    /// Max fee for the transaction. If not provided, max fee will be automatically estimated
+pub struct FeeArgs {
```

Comment:
Why are there 4 instead of 2 args here? From my understanding we only need `fee_token` and `max_fee`

Reply:
Discussed offline. As `v3` has the option to set `gas` and `gas_price`, we should provide it. Also, the `max_fee` flag should be available for `v3`.